### PR TITLE
Ember-CLI Meeting Notes 16 November 2017

### PR DESCRIPTION
**Pull request by hjdivad**

### CODE REVIEW

**Comment by locks** [comment]
RFC unification RFC

File: ember-cli/2017-11/november-16.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# ember-cli Meeting Notes - 2017-11-16
+
+## Attendees
+
+## Agenda
```
Comment (rwjblue) [inline ember-cli/2017-11/november-16.md]: Post agenda items as comments here...

Comment (rwjblue) [reply ember-cli/2017-11/november-16.md]: From @locks: 

* RFC unification RFC

Comment (rwjblue) [reply ember-cli/2017-11/november-16.md]: eslint configuration setup: https://github.com/ember-cli/ember-cli/pull/7443

Comment (locks) [reply ember-cli/2017-11/november-16.md]: oops, and thank you

Comment (rwjblue) [reply ember-cli/2017-11/november-16.md]: ember-cli@2.17.0 release (#soon)

* need blog post
* releases (final beta?)

Comment (rwjblue) [reply ember-cli/2017-11/november-16.md]: Testing in CI on OSX: https://github.com/ember-cli/ember-cli/pull/7440

**Comment by locks** [comment]
Closing due to lack of actual notes. Thanks y'all!

**Pull request closed without merging**